lib/scope.js

```diff
@@ -314,18 +316,20 @@ function define(nockDefs) {
   nockDefs.forEach(function(nockDef) {
     const nscope = getScopeFromDefinition(nockDef)
     const npath = nockDef.path
+    // TODO This line is probably intended to handle the case of a missing
+    // `method`.  However it only works if method is the empty string. Should
+    // we keep this behavior or throw an error?
```

Comment:
I think we should throw an error. It's a pretty vital piece of information and to assume get doesn't feel right.

Reply:
I agree, I run into this before but let it go because it would be a breaking change and there were other things to do. I also think we should enforce setting a method and throw an error if it’s not set.

Reply:
I’d do it in a separate PR though

Reply:
For what it's worth, right now we're already crashing if it's missing, though we do accept an empty string.